siliconcompiler/scheduler/scheduler.py

```
@@ -677,6 +678,15 @@ def __configure_run_setup(self, extra_setup_nodes: Dict[Tuple[str, str], "Projec
                         self.__record.set('status', node_status, step=step, index=index)
         self.__print_status("End - setup")
 
+    @staticmethod
+    def _configure_run_required(task: SchedulerNode) -> Optional[Union[SchedulerFlowReset, SchedulerNodeReset]]:
+        with task.runtime():
+            try:
+                task.requires_run()
+            except (SchedulerNodeReset, SchedulerNodeReset) as e:
```

Ruff (0.14.2)

Exception handler with duplicate exception: `SchedulerNodeReset`

De-duplicate exceptions

(B014)